frontend/pages/_app.tsx

```diff
@@ -2,6 +2,8 @@ import { useEffect } from "react";
 import "semantic-ui-css/semantic.min.css";
 import "../styles/index.css";
 import { SWRConfig } from "swr";
+// eslint-disable-next-line import/no-extraneous-dependencies -- weird, but eslint doesn't like this
```

Comment:
Is this something moving this dep to another section could fix? As you can see, we're doing relative imports above this line and below, so can we try moving this to the top?